Makefile

```diff
@@ -2,11 +2,12 @@ reqs-compile:
 	pip-compile --generate-hashes --output-file requirements/requirements.txt requirements/requirements.in
 	pip-compile --generate-hashes --output-file requirements/requirements.dev.txt requirements/requirements.dev.in
 
+# Note: requirements.dev.txt includes packages from requirements.txt also.
 reqs-download:
-	pip download -r requirements/requirements.txt --dest vendor --platform linux_x86_64 --no-deps
+	pip download -r requirements/requirements.dev.txt --dest vendor --platform linux_x86_64 --no-deps
 
 reqs-install:
-	pip install -r requirements/requirements.txt --no-index --find-links ./vendor/
+	pip install --require-hashes -r requirements/requirements.txt --no-index --find-links ./vendor/
```

Comment:
Unfortunately I think `--require-hashes` will break Amazon Linux 2/Python 3.7 migration if we want to allow side by side deployments without updating the hashes since we now have both versions:

```
- name: "install requirements"
  become_user: "{{ remote_admin_account }}"
  become: yes
  shell: |
    source {{ app_py_virtual_env }}/bin/activate && make reqs-install
  args:
    chdir: "{{ app_pyapp_home }}/{{ common_project_name }}"
  register: out
```

Reply:
@njdister I just removed the `--require-hashes` option for now. Will add a backlog Jira ticket for it.